### PR TITLE
[Docs] Link to colab notebooks

### PR DESCRIPTION
**Pull request by luarss**
@dralabeing @vvbandeira  Link to colab notebooks (ORFS flow + Autotuner flow)

### CODE REVIEW

File: docs/toc.yml [hunk 1]
```diff
@@ -39,3 +39,6 @@ entries:
 
 - file: tutorials/FlowTutorial
   title: Flow Tutorial
```
Comment (dralabeing) [inline docs/toc.yml]: Can you create a separate tutorial for the Colab based execution of GCD under Tutorials? The title says Flow Tutorial. The title should be something like "Running the OpenROAD flow in Google Colab"

File: docs/tutorials/Notebooks.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Colab Quickstart
+
+We provide Google Colab-based notebook packaging for a quickstart to
+OpenROAD-flow-scripts as follows:
+
+1. OpenROAD-flow-scripts automated RTL-GDSII flow for `gcd` design [[link](https://colab.research.google.com/drive/1bfcvqOnbw8q7FmSONLw8q7-yBVvKsQot)]
+1. OpenROAD-flow-scripts Autotuner flow [[link](https://colab.research.google.com/drive/1wye0osn34YVWPvTrfBTftjOfGOtF3ABe)]
```
Comment (dralabeing) [inline docs/tutorials/Notebooks.md]: Numbering. Great to see AutoTuner as part of colab @luarss

**Review by dralabeing — CHANGES_REQUESTED** [review]
Create a separate tutorial for use of ORFS in Colab as indicated.

**Comment by luarss** [comment]
Implemented the changes.

image

image

File: docs/toc.yml [hunk 3]
```diff
@@ -37,5 +37,11 @@ entries:
...
-- file: tutorials/FlowTutorial
-  title: Flow Tutorial
+
+- file: tutorials/TutorialHome
+  title: Tutorials
+  entries:
+  - file: tutorials/FlowTutorial
+    title: Flow Tutorial
+  - file: tutorials/Notebooks
+    title: Running the OpenROAD flow in Google Colab
```
Comment (vvbandeira) [inline docs/toc.yml]:
```suggestion
    title: Running the OpenROAD Flow in Google Colab
```

File: docs/tutorials/Notebooks.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Colab Quickstart
+
+We provide Google Colab-based notebook packaging for a quickstart to
+OpenROAD-flow-scripts as follows:
+
+## OpenROAD-flow-scripts automated RTL-GDSII flow
```
Comment (vvbandeira) [inline docs/tutorials/Notebooks.md]:
```suggestion
## OpenROAD-flow-scripts Automated RTL-GDSII Flow
```

File: docs/tutorials/Notebooks.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+Notebook for the automated flow for `gcd` design in `nangate45` technology node can be found [here](https://colab.research.google.com/drive/1bfcvqOnbw8q7FmSONLw8q7-yBVvKsQot).
+
+## Autotuner Flow
```
Comment (vvbandeira) [inline docs/tutorials/Notebooks.md]:
```suggestion
## AutoTuner Flow
```

File: docs/tutorials/Notebooks.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+Notebook for the OpenROAD-flow-scripts Autotuner flow for `gcd` design in `sky130hd` technology node can be found [here](https://colab.research.google.com/drive/1wye0osn34YVWPvTrfBTftjOfGOtF3ABe).
```
Comment (vvbandeira) [inline docs/tutorials/Notebooks.md]:
```suggestion
Notebook for the OpenROAD-flow-scripts AutoTuner flow for `gcd` design in `sky130hd` technology node can be found [here](https://colab.research.google.com/drive/1wye0osn34YVWPvTrfBTftjOfGOtF3ABe).
```

**Review by vvbandeira — CHANGES_REQUESTED** [review]
Small nits.

File: docs/tutorials/Notebooks.md [hunk 7]
```diff
@@ -3,10 +3,10 @@
 We provide Google Colab-based notebook packaging for a quickstart to
 OpenROAD-flow-scripts as follows:
```
Comment (dralabeing) [inline docs/tutorials/Notebooks.md]: Add this a benefit to using Colab: You can collaborate with your team and easily manage  your OpenROAD design project runs using the Google Colab notebook over the cloud. This helps you to make interactive changes to python or tcl scripts and re-run the flow to examine the impact the changes all from the Notebook.

**Review by dralabeing — CHANGES_REQUESTED** [review]
Add some specific benefits about the use of Google Colab as suggested other than just being a quickstart.

File: docs/tutorials/Notebooks.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
+# Colab Quickstart
+
+Notebook-based packaging offers three main benefits:
+1. Colab is completely free to use, and can be accessed from any web browser.
+1. Colab notebooks can be easily shared with others, making it a great 
+tool for teams to collaborate and manage OpenROAD design runs. 
+1. The notebook can easily be modified to be used for on-premise servers, or 
+notebook-based cloud providers such as LambdaLabs/ Paperspace.
```
Comment (vvbandeira) [inline docs/tutorials/Notebooks.md]:
```suggestion
Notebook-based packaging offers three main benefits:
1. Colab is completely free to use and can be accessed from any web browser.
1. Colab notebooks can be easily shared with others, making it a great 
   tool for teams to collaborate and manage OpenROAD design runs. 
1. The notebook can easily be modified to be used for on-premise servers or 
   notebook-based cloud providers such as LambdaLabs/Paperspace. 
```

File: docs/tutorials/Notebooks.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+tool for teams to collaborate and manage OpenROAD design runs. 
+1. The notebook can easily be modified to be used for on-premise servers, or 
+notebook-based cloud providers such as LambdaLabs/ Paperspace. 
+
+We provide two notebooks for a quickstart to OpenROAD-flow-scripts functionality 
+as follows:
+
+## OpenROAD-flow-scripts Automated RTL-GDSII Flow
+
+Notebook for the automated flow for `gcd` design in `nangate45` technology node can be found [here](https://colab.research.google.com/drive/1bfcvqOnbw8q7FmSONLw8q7-yBVvKsQot).
```
Comment (vvbandeira) [inline docs/tutorials/Notebooks.md]:
```suggestion
The notebook for the automated flow for `gcd` design in `nangate45` technology node can be found [here](https://colab.research.google.com/drive/1bfcvqOnbw8q7FmSONLw8q7-yBVvKsQot).
```

File: docs/tutorials/Notebooks.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+## AutoTuner Flow
+
+Notebook for the OpenROAD-flow-scripts AutoTuner flow for `gcd` design in `sky130hd` technology node can be found [here](https://colab.research.google.com/drive/1wye0osn34YVWPvTrfBTftjOfGOtF3ABe).
```
Comment (vvbandeira) [inline docs/tutorials/Notebooks.md]:
```suggestion
The notebook for the OpenROAD-flow-scripts AutoTuner flow for `gcd` design in `sky130hd` technology node can be found [here](https://colab.research.google.com/drive/1wye0osn34YVWPvTrfBTftjOfGOtF3ABe).
```

**Comment by vvbandeira** [comment]
LGTM, @dralabeing?

**Comment by dralabeing** [comment]
> LGTM, @dralabeing?

LGTM!

**Pull request merged**